src/store/model/amazon.ts

```diff
@@ -22,6 +22,55 @@ export const Amazon: Store = {
 			series: 'test:series',
 			url: 'https://www.amazon.com/dp/B07TDN1SC5'
 		},
+		{
+			brand: 'asus',
+			cartUrl: 'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08L8HPKR6&Quantity.1=1',
+			model: 'dual',
+			series: '3070',
+			url: 'https://www.amazon.com/dp/B08L8HPKR6'
+		},
+		{
+			brand: 'evga',
+			cartUrl: 'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08LW46GH2&Quantity.1=1',
+			model: 'xc3 black',
+			series: '3070',
+			url: 'https://www.amazon.com/dp/B08LW46GH2'
+		},
+		{
+			brand: 'gigabyte',
+			cartUrl: 'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08KY266MG&Quantity.1=1',
+			model: 'gaming oc',
+			series: '3070',
+			url: 'https://www.amazon.com/dp/B08KY266MG'
+		},
+		{
+			brand: 'pny',
+			cartUrl: 'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08HBJB7YD&Quantity.1=1',
+			model: 'xlr8',
+			series: '3070',
+			url: 'https://www.amazon.com/dp/B08HBJB7YD'
+		},
+		{
+			brand: 'pny',
+			cartUrl: 'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08HBJB7YD&Quantity.1=1',
+			model: 'xlr8 uprising',
+			series: '3070',
+			url: 'https://www.amazon.com/dp/B08HBJB7YD'
+		},
```

Comment:
```suggestion
```

Duplicate card from above

Reply:
Resolved, those two cards are on the same product page under different "styles". 🤦‍♂️